console-store/Program.cs

```diff
@@ -0,0 +1,48 @@
+﻿// See https://aka.ms/new-console-template for more information
+using console_store.Clases;
+using System.Collections.Generic;
+
+Console.WriteLine("Hello, World!");
+
+List<Product> products = new List<Product>();
+
+Boolean programWork = true;
+
+while(programWork)
+{ 
+
+    Console.WriteLine("Select action");
+    Console.WriteLine("1 - List of products");
+    Console.WriteLine("2 - Add product");
+    Console.WriteLine("Any button - Exit");
+
+    string inputKey = Console.ReadLine();
+    string inputValue = inputKey.ToString();
```

Comment:
I think you dont need this line

Reply:
Ok, I made the code shorter
`string inputKey = Console.ReadLine().ToString()`;